### ISSUE TITLE
Expected an object for internal function 'functionNameToString' but got '__firstOf__' 

### ISSUE THREAD

**Issue by 2020snc** [Zezombye/overpy#181]
code:
```python
rule "1":
    @Event eachPlayer
    @Condition eventPlayer.isHoldingButton(Button.INTERACT)
    del eventPlayer.A[0]
```

overpy version: 5.7.3

![ScreenShot_2020-06-23_21-45-00](https://user-images.githubusercontent.com/64687143/85411980-a457d780-b59b-11ea-8e74-06c6dfb96559.png)

**Comment by Zezombye** [comment]
Thanks for reporting, I'll try to fix that soon!

In the meanwhile you can use `[null]` or the `#!disableOptimizations` directive.

**Comment by Zezombye** [comment]
Fixed in 5.7.4
